README.md

```diff
@@ -78,9 +71,11 @@ The web component can be included in a page by using the `<editor-wc>` HTML elem
 - `embedded`: Enable embedded mode which hides some functionality (defaults to `false`)
 - `output_split_view`: Start with split view in output panel (defaults to `false`, i.e. tabbed view)
```

Comment:
🌱 I _think_ there's more attributes than this now 👀

Reply:
Ohh yes, I was planning to come back to fill this in and I forgot, thanks!